### PR TITLE
Updated endpoints

### PR DESCRIPTION
**Pull request by michielgerritsen**
Hi,

I've updated some of the endpoints because my api key is bound to those. I was unable to use this library with older endpoints. I will update/add more pull requests as i hit more bumps.

### CODE REVIEW

File: src/ComplexTypes/BaseLocation.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ class BaseLocation extends BaseType
     /**
      * @var string
      */
-    protected $OpeningTime = null;
+    protected $OpeningTime = '00:00:00';
```
Comment (michielgerritsen) [inline src/ComplexTypes/BaseLocation.php:24]: According to the documentation, `OpeningTime` is optional. But when you trigger `getNearestLocations` you will get an exception stating that null is not allowed. That's why i default to `00:00:00`.

Comment (ameenross) [reply src/ComplexTypes/BaseLocation.php:24]: This is a confirmed bug on PostNL's side.

**Comment by ameenross** [comment]
I find it strange that tokens would limit the *version* of endpoints. To the best of my knowledge they only limit the type of endpoint.

In any case, this just fakes support for the new version, as the new version has schema changes. More needs to be changed to actually support the new version.

**Comment by ameenross** [comment]
By the way, you can already instantiate a client with a custom WSDL url. If you want, you can also pass it to the wrapping class like so:

```php
$client = new Postnl(/* stuff */);
$ddClient = new DeliveryDateClient($apikey, false, 'https://api.postnl.nl/shipment/v2_2/calculate/date/soap.wsdl');

$client->setDeliveryDateClient($ddClient);
```

**Pull request closed without merging**